src/js/components/AppComponent.jsx

```diff
@@ -26,6 +26,19 @@ var AppComponent = React.createClass({
     );
   },
 
+  getStatus: function () {
+    var model = this.props.model;
+    var status = "Running";
+
+    if (model.deployments.length > 0) {
+      status = "Deploying";
+    } else if (model.instances === 0 && model.tasksRunning === 0) {
```

Comment:
I see! I was just unsure if 0 instances should still be considered "Suspended" or not – thanks for clarifying!